src/app/_shared/_components/header/header.component.html

```diff
@@ -1,4 +1,15 @@
 <div class="container text-center my-4">
-    <h1 class="display-4 font-monospace" style="font-variant: small-caps;">Ketal</h1>
-    <p class="lead">Grosse Guinze Generator</p>
+<form [formGroup]="languageForm">
+  <div class="row position-absolute top-0 end-12">
+    <label for="language-select" class="col-form-label col-5" [translate]="'Label_Language'"></label>
+    <div class="col-7">
```

Comment:
col 7 to small 
![image](https://github.com/lammensmichel/ketal/assets/66747756/9f5a1f5a-b598-4a56-a82b-d1510889967f)

Reply:
Changed, should be alright now